examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -215,6 +217,8 @@ void ApplicationInit()
 #endif
     Clusters::TemperatureControl::SetInstance(&sAppSupportedTemperatureLevelsDelegate);
 
+    IcdManagementServer::GetInstance().SetSessionKeyStore(Server::GetInstance().GetSessionKeystore());
```

Comment:
Will this cause problems if the all-clusters app is built with `CHIP_CONFIG_ENABLE_ICD_SERVER`?
Also, can you add the issue to the code?

Reply:
No it won't worst case a variable will be set twice with the same value.